src/sql/workbench/contrib/notebook/test/electron-browser/cell.test.ts

```diff
@@ -349,11 +349,16 @@ suite('Cell Model', function (): void {
 		};
 		let cell = factory.createCell(mdCellData, undefined);
 		assert(cell.isCollapsed === false);
+		assert(cell.isParameter === false);
+
 		cell.isCollapsed = true;
+		cell.isParameter = true;
 		// The typescript compiler will complain if we don't ignore the error from the following line,
 		// claiming that cell.isCollapsed will return true. It doesn't.
 		// @ts-ignore
 		assert(cell.isCollapsed === false);
+		// @ts-ignore
```

Comment:
Is the ts-ignore necessary here too?

Reply:
Yes, the same problem occurs for cell.isCollapsed. I can edit the comment to inform both cell.isCollapsed and cell.isParamterer.